src/lib.rs

```diff
@@ -38,8 +40,15 @@ pub struct DiscordRpcPlugin {
 
 #[deny(non_snake_case)]
 impl Plugin for DiscordRpcPlugin {
-    fn new(plugin_data: &PluginData) -> Self {
-        plugin_data.register_sq_functions(presence::fetch_presence);
+    const PLUGIN_INFO: PluginInfo = PluginInfo::new(
+        "DISCORDRPC\0",
+        "DISCORDXD\0",
```

Comment:
I know `log_name` being shorter is so it matches the rest in length but why the `XD` suffix. Just cause `xD` or am I missing some obvious abbreviation here? ^^


If not, you could also do something like `DSCRD-RPC` ig

Reply:
I just couldn't a come up with anything better better than `XD` suffix.
`DSCRD-RPC` is definitely better.